### PR TITLE
fix(textField): add responsivity to textfield

### PR DESCRIPTION
**Pull request by lolottetheclash**
This PR implements these following specifications : https://github.com/okp4/ui/issues/209

### CODE REVIEW

File: src/ui/atoms/textField/TextField.tsx [hunk 1]
```diff
@@ -53,12 +58,14 @@ export const TextField: React.FC<TextFieldProps> = ({
   value,
   defaultValue,
   helperText,
-  inputRef
+  inputRef,
+  fullWidth = false
 }: TextFieldProps): JSX.Element => {
   const inputClass = classNames(`okp4-text-field-core `, { error: hasError })
+  const containerClass = classNames(`okp4-text-field-main ${size}`, { 'full-width': fullWidth })
```
Comment (fredericvilcot) [inline src/ui/atoms/textField/TextField.tsx:65]: You could use here shortcut syntax for object: 
```suggestion
  const containerClass = classNames(`okp4-text-field-main ${size}`, { fullWidth })
```

**Review by fredericvilcot — CHANGES_REQUESTED** [review]
That's a good catch for your first PR! 🚀
I just wrote a very minor remark, it's up to you!

File: src/ui/atoms/textField/TextField.tsx (hunk 1 above)
Comment (fredericvilcot) [reply src/ui/atoms/textField/TextField.tsx:65]: My bad 😜

**Review by fredericvilcot — DISMISSED** [review]
Great!

**Review by ErikssonJoakim — DISMISSED** [review]

**Review by ccamel — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**